app/src/main/java/co/co/nimblehq/showcases/poly/place/ui/common/StartDrawableText.kt

```diff
@@ -0,0 +1,56 @@
+package co.co.nimblehq.showcases.poly.place.ui.common
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.TextUnit
+import co.co.nimblehq.showcases.poly.place.R
+import co.co.nimblehq.showcases.poly.place.ui.theme.AppTheme.dimensions
+
+@Composable
+fun StartDrawableText(
+    modifier: Modifier = Modifier,
+    iconSize: Dp = dimensions.iconSizeNormal,
+    @DrawableRes iconResId: Int,
+    iconTint: Color = Color.White,
+    drawablePadding: Dp = dimensions.spacingSmall,
+    textColor: Color = Color.White,
+    textSize: TextUnit = dimensions.textSizeNormal,
+    text: String? = null
+) {
+    Row(
+        modifier = modifier,
+        verticalAlignment = Alignment.CenterVertically
```

Comment:
Please update to `Alignment.Top` as it will keep align on the top in case the address is long and run into multiple lines:
```suggestion
        verticalAlignment = Alignment.Top
```

Currently, it shows on the middle of the text.

![Screenshot 2566-03-07 at 1 29 08 PM](https://user-images.githubusercontent.com/12026942/223338849-a99b3f8b-4b49-4d25-a3fa-4c81c0d411d7.png)